tutorials/rendering/renderers.rst

```diff
@@ -0,0 +1,315 @@
++---------------------+--------------------------+--------------------------+--------------------------+
+| Desktop             | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Mobile              | ✔️ Yes (low-end).        | ✔️ Yes (high-end).       | ❌ Supported, but poorly |
+|                     |                          |                          | optimized. Use Mobile or |
+|                     |                          |                          | Compatibility instead.   |
++---------------------+--------------------------+--------------------------+--------------------------+
+| XR                  | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
+|                     |                          |                          |                          |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Web                 | ✔️ Yes.                  | ❌ No.                   | ❌ No.                   |
++---------------------+--------------------------+--------------------------+--------------------------+
+| 2D Games            | ✔️ Yes.                  | ✔️ Yes, but              | ✔️ Yes, but              |
+|                     |                          | Compatibility is usually | Compatibility is usually |
+|                     |                          | good enough for 2D.      | good enough for 2D.      |
++---------------------+--------------------------+--------------------------+--------------------------+
+| 3D Games            | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
++---------------------+--------------------------+--------------------------+--------------------------+
+| **Feature set**     | 2D and core 3D features. | Most rendering features. | All rendering features.  |
++---------------------+--------------------------+--------------------------+--------------------------+
+| 2D rendering        | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
+| features            |                          |                          |                          |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Core 3D rendering   | ✔️ Yes.                  | ✔️ Yes.                  | ✔️ Yes.                  |
+| features            |                          |                          |                          |
++---------------------+--------------------------+--------------------------+--------------------------+
+| Advanced            | ❌ No.                   | ✔️ Yes, limited by       | ✔️ Yes. All rendering    |
+| rendering features  |                          | mobile hardware.         | features are supported.  |
```

Comment:
Use warning triangle icon for Mobile, since features like SDFGI and volumetric fog are missing:

```suggestion
| Advanced            | ❌ No.                   | ⚠️ Yes, limited by       | ✔️ Yes. All rendering    |
| rendering features  |                          | mobile hardware.         | features are supported.  |
```